testing/run_deploy_app.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# This script shows how to manually trigger a load test for 1-click-deployment service.
+#
+# All the projects created for load tests have one owner
+# "load-test-owner@kf-gcp-deploy0.iam.gserviceaccount.com".So in order to run the load
+# test you have to set GOOGLE_APPLICATION_CREDNTIALS and SERVICE_CLIENT_ID both point to
+# service account "load-test-owner@kf-gcp-deploy0.iam.gserviceaccount.com"
+# https://pantheon.corp.google.com/iam-admin/serviceaccounts/details/112401461927766705527?organizationId=714441643818&project=kf-gcp-deploy0
+#
+# CLIENT_ID and CLIENT_SECRET both point to
+# https://pantheon.corp.google.com/apis/credentials/oauthclient/459682233032-a76ps35eh3j8odvudf4292bgq0jam74i.apps.googleusercontent.com?project=kf-gcp-deploy0&organizationId=714441643818
+
+GOOGLE_APPLICATION_CREDENTIALS=/usr/local/google/home/zhenghui/env/kf-gcp-deploy0-load-test.json \
+SERVICE_CLIENT_ID=112401461927766705527 \
+CLIENT_ID=459682233032-a76ps35eh3j8odvudf4292bgq0jam74i.apps.googleusercontent.com \
+CLIENT_SECRET=eEWmsMuOo7s1o04bRctfehuD \
```

Comment:
CLIENT_SECRET should be passed as Env Var and not publicly available.

Reply:
Thanks for catching this security issue. I reset the client secret.
This is the env var for running a python cmd. I just change it to avoid putting the actual value here.